src/read_write/lg_interface.jl

```diff
@@ -0,0 +1,48 @@
+LightGraphs.nv(g::AbstractGraph{V,E}) where {V,E} = num_nodes(g)
+LightGraphs.ne(g::AbstractGraph{V,E}) where {V,E} = num_edges(g)
+
+function LightGraphs.outneighbors(g::AbstractStaticGraph{NT,E}, v::V) where {V, NT<:AbstractNode{V},E}
```

Comment:
This should be
```
LightGraphs.outneighbors(g::AbstractStaticGraph{V}, v::V)
```

Reply:
I guess you will want the two:
```
LightGraphs.outneighbors(g::AbstractStaticGraph{NT,E}, v::V) where {V, NT<:AbstractNode{V},E}
```
means you can call `LightGraphs.outneighbors(g,"a")` to get neighbors for the node with key "a" while 
```
LightGraphs.outneighbors(g::AbstractStaticGraph{V,E}, v::V) where {V,E}
```
allows for `LightGraphs.outneighbors(g, nodes(g)[1])`

I added the two ways for both `in` and `out`

Reply:
Yes, great